RedHatInsights/vulnerability-ui
# fix(VULN-2757) - Global filter works on refresh

[VULN-2757](https://issues.redhat.com/browse/VULN-2757)

How to test:
1. Open systems tab
2. Apply global filter
3. Refresh
4. Verify that filter still appears in the UI
5. Repeat for CVE Detail
